src/System.ServiceModel.Security/tests/ServiceModel/SecurityKeyTypeTest.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System.IdentityModel.Tokens;
+using Infrastructure.Common;
+using Xunit;
+
+public static class SecurityKeyTypeTest
+{
+    [WcfFact]
+    public static void Get_EnumMembers_Test()
```

Comment:
Our convention is to not use the word "test" as part of the name of our test methods. The reason for this is that in reporting on test passes the namespace is included as part of the full test name, so this test would read as:  `SecurityKeyTypeTest.Get_EnumMembers_Test`
Instead we want it to be `SecurityKeyTypeTest.Get_EnumMembers`